src/style/global.sass

```
@@ -41,8 +44,10 @@ ul, ol
 code
     font-family: $font-code
     padding: 1rem
+    box-sizing: border-box
     color: $code-default-color
     unicode-bidi: plaintext
     display: block
     background: $bg-color-secondary
     border-radius: 1rem
+    overflow: scroll
```

Shouldn't displayed always as `scroll`. Only on the X axis, and only when content is overflowed.

```suggestion
    overflow-x: auto
```